tests/fixtures/fixtures_job.py

```diff
@@ -64,43 +73,43 @@ def job_mock(auth_mock, requests_mock):
 
 
 @pytest.fixture()
-def job_live(auth_live):
+def job_live(auth_live, project_id_live):
```

Comment:
@javidq you defined the fixture `project_id_live` in fixtures_auth.py
I thought only the fixtures defined in `conftest.py` can be shared across all tests. How does it work without explicit import?

Reply:
@seedlit I missed that, thanks, but I hope the pipeline failures will detect those